src/widgets/helper/ChannelView.hpp

```diff
@@ -179,6 +179,9 @@ class ChannelView final : public BaseWidget
 
     LimitedQueueSnapshot<MessageLayoutPtr> &getMessagesSnapshot();
 
+    // Returns true if message should be included
```

Comment:
This should mention that `m` must "logically" belong to this channel, or rather that the origin of `m` (the channel it originates from or smth) isn't checked.

Reply:
I just moved the function from private to public part, realistically I do not like the name that was used for it, I was searching for a function like "isMessageFiltered" or "filterMessage", both of these names better describe what the function does in my opinion, though shouldIncludeMessage was the name that was already there.

Reply:
so far I just renamed the m to message, the other naming way would require a logic flip everywhere where its used.
( isMessageFiltered == not shouldIncludeMessage)

Reply:
is no longer needed to be public, so reverted